src/libsync/owncloudpropagator.h

```diff
@@ -215,6 +219,8 @@ class OWNCLOUDSYNC_EXPORT PropagateDirectory : public PropagatorJob {
     virtual void abort() Q_DECL_OVERRIDE {
         if (_firstJob)
             _firstJob->abort();
+        if (_bundleJob)
+            _bundleJob->abort();
```

Comment:
is that necessary,  shouldn't the _bundleJob be in the _subjobs if it was started?